charmcraft.yaml

```diff
@@ -10,6 +10,7 @@ bases:
       channel: "20.04"
 parts:
   charm:
+    charm-binary-python-packages: [cryptography, jsonschema]
```

Comment:
We should not be doing this in production code. Out charms are supposed to be built from source.

(This is the blocking comment)

Reply:
You are right, I forget this